packages/env/test/github.test.js

```diff
@@ -62,5 +63,10 @@ describe('GitHub', () => {
       expect(env).toHaveProperty('commit', 'gh-env-sha');
       expect(env).toHaveProperty('branch', 'gh-env-branch');
     });
+
+    it('has the correct parallel nonce in 2.x', () => {
```

Comment:
What is `2.x` in this context?

Reply:
You know when I ripped it from CircleCI, I thought it was referring to the client. But I think it means circleci 2? No tests for 1.x in circle tho. 

But yea, removed, it doesn't make sense.